website/docs/guides/continuos_validation_with_terraform_cloud.markdown

```diff
@@ -0,0 +1,72 @@
+
+Below, this guide shows examples of how data returned from the New Relic provider can be used to define checks in your Terraform configuration. For more information about continuous validation visit the [Workspace Health](https://developer.hashicorp.com/terraform/cloud-docs/workspaces/health#continuous-validation) page in the Terraform Cloud documentation.
+
+## Example - Check the configuration of a Notification Destination (`newrelic_notification_destination`)
+
+[Notification destinations](https://docs.newrelic.com/docs/alerts-applied-intelligence/notifications/destinations/#destinations-and-notifications-statuses) configured with the New Relic provider expose multiple attribute that indicate the configuration and status of the destination.
+
+Continuous validation can be used to assert the configuration and status of a notification
+destination and detect if there are any unexpected status changes that occur out-of-band.
+
+The example below shows how a check block can be used to assert that a
+notification destination is configured and working as expected.
+
+```hcl
+data "newrelic_notification_destination" "example" {
+  id = "1e543419-0c25-456a-9057-fb0eb310e60b"
+}
+
+check "check_notification_destination_status" {
+  assert {
+    condition = data.newrelic_notification_destination.example.type == "SLACK"
+    error_message = format("Notification destination (%s) type should be set to 'SLACK', instead type is '%s'",
+      data.newrelic_notification_destination.example.name,
+      data.newrelic_notification_destination.example.type
+    )
+  }
+  assert {
+    condition = data.newrelic_notification_destination.example.status == "DEFAULT"
+    error_message = format("Notification destination (%s) should be in a 'DEFAULT' status, instead status is '%s'",
+      data.newrelic_notification_destination.example.name,
+      data.newrelic_notification_destination.example.status
+    )
+  }
+}
+```
+
+## Example - Check the last updated time for Synthetic Secure Credential (`newrelic_synthetics_secure_credential`)
+
+[Secure credentials](https://docs.newrelic.com/docs/synthetics/synthetic-monitoring/using-monitors/store-secure-credentials-scripted-browsers-api-tests/) configured with the New Relic provider expose a `last_updated` attribute that indicates when the credential was last updated.
+
+You can use this attribute to validate the age of the secure credential and make sure it's either set to latest, or not updated without permission.
+
+```hcl
+data "newrelic_synthetics_secure_credential" "example" {
+  key = "MY_KEY"
+}
+
+check "check_synthetics_secure_credential" {
+  assert {
+    condition = timecmp(timestamp(), timeadd(data.newrelic_synthetics_secure_credential.example.last_updated, "-720h")) < 0
```

Comment:
I think this condition needs to be 
```
timecmp(timestamp(), timeadd(data.newrelic_synthetics_secure_credential.example.last_updated, "+720h")) < 0
```

If the current timestamp is within 30 days of the last update, the credential is still valid. If this condition fails, the error listed below needs to be thrown.

(I tried this with two credentials - one that I just created, and one that was last updated in March 2023, and this now works fine, unless I'm missing something elementary - apologies if that is the case)